src/App.tsx

```diff
@@ -1,7 +1,9 @@
-import React from 'react';
+import { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
+import { Tab } from './types/Tab';
```

Comment:
Add type file to the PR
![image](https://user-images.githubusercontent.com/91411692/235893816-e7c44c85-d57f-43de-8fe7-f89705b2d44f.png)